keyboards/kprepublic/bm80v2/readme.md

```diff
@@ -0,0 +1 @@
+# bm80v2
```

Comment:
This needs to be filled out, RE:
https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md